src/pseudopeople/configuration.py

```diff
@@ -0,0 +1,203 @@
+from pathlib import Path
+from typing import Dict, NamedTuple, Union
+
+import yaml
+from vivarium.config_tree import ConfigTree
+
+from pseudopeople.schema_entities import FORMS
+
+
+class Keys(NamedTuple):
+    """NamedTuple containing all key names used in the configuration file
+    NOTE: 'additional_parameters' is actually a dict with its own key-values defined
+    """
+
+    ROW_NOISE = "row_noise"
+    OMISSION = "omission"
+    DUPLICATION = "duplication"
+    COLUMN_NOISE = "column_noise"
+    ROW_NOISE_LEVEL = "row_noise_level"
+    TOKEN_NOISE_LEVEL = "token_noise_level"
+    ADDITIONAL_PARAMETERS = "additional_parameters"
+
+
+# Define non-baseline default items
+# NOTE: default values are defined in entity_types.RowNoiseType and entity_types.ColumnNoiseType
+DEFAULT_NOISE_VALUES = {
+    # TODO: Uncomment when omission gets implemented
```

Comment:
I commented these out b/c we don't actually have row functions implemented yet. I think this would work fine if they were not commented out, but I'd rather the config file simply not have them there at all if they're unused (kinda like how we're not adding a column function to the config until it's actually implemented)

Reply:
There shouldn't be an engineering reason to have these commented out or test for `is_implemented` is there? The actual noise functions are existent even if they aren't doing anything.

Reply:
It was simply to keep in line w/ how we have not been adding column noising functions to the config file until the functions were actually implemented. And yet we've always had these duplications and omissions values in there despite those two functions not existing - I never liked that

Reply:
There was never a reason to do that though... Making code that will be (or is) vestigial is bad.

Reply:
You're referring to the is_implemented stuff? I can remove that but then need to remove a bunch of code that will later jsut be added back

Reply:
The is_implemented stuff is completely unnecessary programmatically now. Why would it be needed later?

Reply:
Sure, idc that much. I do not like the idea of generating the config file with unimplemented stuff but it's not a hill I'm going to die on. So for this sort of pre-approval check go ahead and ignore that. Once I get signoff on the overall approach I'll implement everything else.